README.md

```diff
@@ -86,6 +86,11 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 
 - [vite-tsconfig-paths](https://github.com/aleclarson/vite-tsconfig-paths) - Support for TypeScript's path mapping.
 
+
+### Markdown
+
+- [vite-plugin-vuedoc](https://github.com/JasKang/vite-plugin-vuedoc) - A markdown & vue preview plugin for vite.
```

Comment:
Can you move this to the "Loaders" section? And you can omit the `for vite` statement.

Reply:
ok, thank u